README.md

```diff
@@ -1,8 +1,37 @@
-# DataJoint interactive tutorial with GitHub Codespace
+# Welcome to DataJoint tutorials!
 
-Interactive tutorials on the DataJoint framework, in Python. Throughout this set of tutorials, you will learn
+DataJoint is an open-source tool library to design, build and automate data analysis, pipelines, and data sharing for neuroscience experiments and research labs. 
 
-- DataJoint basics (~1 hour)
+This document will guide you as a new DataJoint user through interactive tutorials organized in [Jupyter notebooks](https://jupyter-notebook.readthedocs.io/en/stable/) and written in [Python](https://www.python.org/).
+
+*Please note that these hands-on DataJoint tutorials are friendly to non-expert users, and advanced programming skills are not required.* 
+
+## Table of contents 
+- Notebooks: interactive notebooks to be a master of DataJoint. The Calcium Imaging and Electrophysiology tutorials are common examples of data structure and analysis. In addition, some fill-in-the-blank sections are included for you to code yourself!
+    - 01-DataJoint Basics
+    - 02-Calcium Imaging Imported Tables
+    - 03-Calcium Imaging Computed Tables
+    - 04-Electrophysiology Imported Tables
+    - 05-Electrophysiology Computed Tables
+
+- Completed exercises: notebooks with the code sections completed and solved.
+
+- Short tutorials:
+    - DataJoint in 30min
+    - University
+
+- Tutorial pipeline
+    - Ephys cell activity
+    - Imaging
+    - Mouse session
+
```

Comment:
Since we decided to merge the contents of these files into the notebooks for simplicity.
```suggestion
```